MHIdentityKit/OAuth2/Flows/AuthorizationCodeGrantFlow.swift

```diff
@@ -310,6 +321,97 @@ open class AuthorizationCodeGrantFlow: AuthorizationGrantFlow {
             return true
         }
     }
+        
+    @available(iOS 13, tvOS 13.0.0, macOS 10.15, *)
+    open func authenticate() async throws -> AccessTokenResponse {
```

Comment:
This implementation not needed, because it is handled by the `AuthorizationGrantFlow` extension.